roles/telemetry_autoscaling/tasks/creating_stack.yml

```diff
@@ -101,20 +101,19 @@
 - name: RHOSO-12653 Verify physical_resource_id match the alarm id for cpu_alarm_low
   ansible.builtin.shell: |
     # source ~/overcloudrc;
-    {{ openstack_cmd }} alarm list |grep -i cpu_alarm_low | awk '{print $2}'
+    {{ openstack_cmd }} alarm list |grep -i cpu_alarm_low | grep {{ stack_name }} | awk '{print $2}'
```

Comment:
I added a grep for the stack_name. This is just to allow to run these tests when there is already an autoscaling stack with a different name present in the cluster.